src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -110,12 +110,12 @@ public override Task<bool> HandleRequestAsync()
             var remoteSignOutContext = new RemoteSignOutContext(Context, Scheme, Options, message);
             await Events.RemoteSignOut(remoteSignOutContext);
 
-            if (remoteSignOutContext.HandledResponse)
+            if (remoteSignOutContext.State == EventResultState.HandleResponse)
             {
                 Logger.RemoteSignOutHandledResponse();
                 return true;
             }
-            if (remoteSignOutContext.Skipped)
+            if (remoteSignOutContext.State == EventResultState.SkipToNextMiddleware)
```

Comment:
So looking at the current names,

HandleRequestContext => BaseControlContext since this isn't actually something that all RemoteAuthenticationHandlers use.

RemoteAuthenticateResultContext => AuthenticateResultControlContext which is only used by OIDC/JwtBearer events.